.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,71 @@
+name: Bug Report
```

Comment:
Should we add a "how to reproduce" section or something like that? Perhaps a link to a gist/repo or something along those lines.